src/main/java/backend/model/event/Team.kt

```diff
@@ -52,8 +52,8 @@ class Team : BasicEntity {
     @OneToMany(cascade = arrayOf(ALL), mappedBy = "team", orphanRemoval = true)
     private var invitations: MutableList<Invitation> = ArrayList()
 
-    @OneToOne(cascade = arrayOf(ALL), orphanRemoval = true, fetch = LAZY)
-    lateinit var profilePic: Media
+    @ManyToOne(fetch = LAZY)
+    var profilePic: Media? = null
```

Comment:
Why many to one? `profilePic 1-->1 team` i'd say. One profile pic belongs to one team, and each team has only one profile picture?